components/ArticlePageLayout.js

```diff
@@ -88,6 +88,7 @@ const useStyles = makeStyles(theme => ({
   },
   articleList: {
     padding: 0,
+    marginBottom: -12,
```

Comment:
Sorry, I forgot to consider the case when there is no "next" button:

![image](https://user-images.githubusercontent.com/108608/81596572-e4ea0000-93f6-11ea-9841-4d63cce39514.png)

(It's fine in mobile, though; it's just that floating action buttons are covering the last item)